Source/ProjectFirma.Web/Views/FirmaDhtmlxGridHtmlHelpers.cs

```diff
@@ -62,5 +62,29 @@ public static HtmlString DhtmlxGrid<T>(this HtmlHelper html, GridSpec<T> gridSpe
 
             return new HtmlString(dhtmlxGrid);
         }
+
+        /// <summary>
+        /// Temp method creates grids that exclude a download filtered grid option for 1992

```

Comment:
Is this temp method still used? 
I would remove the word temp. Add a date and your initials and then put the full card number in there(WADNR-1992). I was a lil confused at first by just 1992.